catalog/ui/catalog-ui-search/src/main/webapp/js/CQLUtils.js

```diff
@@ -31,6 +31,14 @@ define([
                 });
             }
 
+            //sanitize multipolygons
+            let multipolygons = cqlString.match(/'MULTIPOLYGON\(\(\(.*\)\)\)'/g);
```

Comment:
doesn't multipolygon have a blank space after "MULTIPOLYGON"?

Reply:
On the front end this comes in without the space